composer.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "wordpress/twentytwentyfive",
```

Comment:
Should we include the PHP min version in this file?

Reply:
I think we should, since the theme’s minimum requirement is PHP 7.2.24+. Should we add PHP 7.2.24+?